src/test/java/UploadReceiptServletTest.java

```diff
@@ -214,11 +219,21 @@ public void doPostUploadsReceiptToDatastoreLiveServer() throws IOException {
     Assert.assertEquals(receipt.getProperty("timestamp"), PAST_TIMESTAMP);
     Assert.assertEquals(receipt.getProperty("categories"), CATEGORIES_COLLECTION);
     Assert.assertEquals(receipt.getProperty("userId"), USER_ID);
+
+    String response = stringWriter.toString();
+    String expectedResponse = createReceiptEntity(IMAGE_URL, PRICE, STORE, RAW_TEXT, BLOB_KEY,
+        PAST_TIMESTAMP, CATEGORIES_COLLECTION, USER_ID);
+    Assert.assertTrue(response.contains(expectedResponse));
   }
 
   @Test
   public void doPostUploadsReceiptToDatastoreDevServer() throws IOException {
     helper.setEnvIsLoggedIn(true);
+
+    StringWriter stringWriter = new StringWriter();
```

Comment:
Looks like this setup is done in every test, so I would put this setup in the @Before method